miio/airqualitymonitor.py

```diff
@@ -206,11 +241,12 @@ def __init__(
 
         if model in AVAILABLE_PROPERTIES:
             self.model = model
-        else:
+        elif model is not None:
             self.model = MODEL_AIRQUALITYMONITOR_V1
```

Comment:
When a model is provided, but not in available_properties fallback is still there.